app/controllers/raw_records_controller.rb

```diff
@@ -1,9 +1,31 @@
 # frozen_string_literal: true
 
 class RawRecordsController < ApplicationController
+  before_action :find_fields, only: [:index]
+  before_action :find_transformation_definition, only: [:index]
+
   def index
-    transformation_definition = TransformationDefinition.find(params[:transformation_definition_id])
+    record = @transformation_definition.records(params[:page].to_i)[params['record'].to_i - 1]
```

Comment:
do we need a spec for this?

Reply:
I don't think so bro, it's just accounting for the difference between page numbers and array indexes :)